.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: node_js
+
+node_js:
+  - '6'
+
+dist: trusty
+sudo: required
```

Comment:
Do we need to use `dist` and `sudo` (["The Trusty beta Build Environment"](https://docs.travis-ci.com/user/trusty-ci-environment/)) for any specific reason?

I'm under the impression that we could do without.